### ISSUE TITLE
Publish publicly to Play Store

### ISSUE THREAD

**Issue by jbphet** [phetsims/quake#22]
One of my "rocks" for this quarter is to publish the version that was essentially completed last quarter to the Google Play Store.

**Comment by jbphet** [comment]
@mattpen - You're the only one on the team (that I know of anyway) who has experience with publishing Android apps to the Play Store, since you've done it for the PhET Android app.  Is there anything that I should know about the process?  I'm happy to set up a meeting if you'd like.  The Haptics Playground app is currently in the "Internal Testing" state, so I'm starting from there and moving it towards publication.

**Comment by mattpen** [comment]
I can't think of anything specific. I'd be happy to join a call and talk about the process. @oliver-phet has been pretty involved with the play store management of the phet app as well.

**Comment by jbphet** [comment]
I published version 1.1.0 for alpha testing by the QA team, see https://github.com/phetsims/qa/issues/842.

**Comment by jbphet** [comment]
I published a 1.1.1 version for Beta Testing, and after QA finished that testing we decided to move forward with publishing the app to the Play Store.  I've taken what I believe are the appropriate steps using the Play Store Console, and it is now showing the status as "In Review".  It looks like this:

![image](https://user-images.githubusercontent.com/3082831/203186067-f8500fe8-52e4-410d-8679-e1749145b24d.png)

I'll check this periodically and hope to see it soon on the store.
